201-representing-CNAB-in-OCI.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Representing CNAB bundles in OCI Registries
+weight: 201
+---
+
+# Representing CNAB bundles in OCI Registries
+
+This section describes how the bundle file and the information represented in the bundle MUST be represented when distributing bundles using OCI registries.
```

Comment:
suggestion to avoid saying represented multiple times, feel free to ignore
```suggestion
This section describes how the bundle file and the information contained in the bundle MUST be represented when distributing bundles using OCI registries.
```